### ISSUE TITLE
Other search engine (Bing, Ebay, etc..) classes are not loaded

### ISSUE THREAD

**Issue by ZilvinasKucinskas** [serpapi/google-search-results-ruby#7]
## Script to reproduce

```
$ gem install google_search_results
$ irb
> require 'google_search_results'
> BingSearchResults
NameError (uninitialized constant BingSearchResults)
```

## Problem

I think [this is a good explanation on Rubygems Guides](https://guides.rubygems.org/patterns/#loading-code). I believe changing the structure of the `/lib` should be needed to fix the issue.

**Comment by jvmvik** [comment]
Thanks for reporting this. I'm working on a fix. ETA 24h
  It looked like that Ruby / Gem 2.6+ is affected..

**Comment by ilyazub** [comment]
This was fixed in d85bdc950007561fdd3b60fc51c723336dd9caa0. @jvmvik please correct me if I'm wrong.
